FirebaseSessions/Sources/ApplicationInfo.swift

```diff
@@ -63,4 +75,12 @@ class ApplicationInfo: ApplicationInfoProtocol {
   var mccMNC: String {
     return FIRSESValidateMccMnc(networkInfo.mobileCountryCode, networkInfo.mobileNetworkCode) ?? ""
   }
+  
+  var environment: DevEnvironment {
+    let environment = envParams["FIREBASE_RUN_ENVIRONMENT"]
+    if (environment != nil) {
+      return DevEnvironment(rawValue: environment!.trimmingCharacters(in: .whitespaces).lowercased()) ?? DevEnvironment.prod
```

Comment:
Can we change `environment!` to `environment?` - Using an `!` can crash the app if it's null

Reply:
With a nil check above, it is safe to force unwrap it here. No necessary risks here.